Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN git clone --single-branch --depth 1 https://github.com/kubernetes-client/c
 RUN git clone --single-branch --depth 1 --branch v4.2-stable https://libwebsockets.org/repo/libwebsockets
 RUN git clone --single-branch --depth 1 --branch release/0.2.5 https://github.com/yaml/libyaml
 RUN git clone --single-branch --depth 1 https://github.com/antlr/antlr4.git
+RUN git clone --single-branch --depth 1 https://github.com/erikmuttersbach/libhdfs3.git
```

Comment:
We need to depend on a particular release number. For example, in line 31 `--branch release/0.2.5` which specifically sets the version to use. We should not depend on the master branch as it keeps on changing and there is a chance of breaking at some point.

Reply:
There is only one version(tag) in this repo, which was done in 2015. and when I use that version and try to install hdfs dependencies it gives me an error regarding an index out of bound error. I follow the exact same instructions that they mentioned for installing dependencies. 

Also when installing dependencies without using that version/tag, it works properly. I think this version is very older one and now not working properly. Therefore I think it is better not to use that version/tag for installing HDFS dependencies here.